source/hands_on/rainbow.rst

```diff
@@ -106,6 +106,25 @@ The network interface Rainbow used is defined as follows:
 The Benchmark result of Rainbow implemented in DI-engine is shown in `Benchmark <../feature/algorithm_overview_en.html>`_
 
 
+
+Experiments on Rainbow Tricks
+-----------------------------
+We conduct experiments on lunarlander environment using rainbow(dqn) policy, comparing the performance of dueling, priority, and priority_IS tricks with baseline. 
+
+1. ``reward_mean`` over ``iteration`` is used as an evaluation metric.
+   
+2. Each experiment setting is done for three times with random seed 0, 1, 2 and average the results to insure stochasticity.
+
+3. By setting the ``exp_name`` in config file, the experiement results can be saved in specified path. Otherwise, it will be saved in ``./default_experiment``.
+
+The result is shown in the figure below. As we can see, with tricks on, the speed of convergence is increased by a large amount. In this experiement setting, dueling trick contributes most to the performance. 
+
+.. image:: 
```

Comment:
You need to explain the meaning of each experiment, and add code links for corresponding options.

Reply:
done